pkg/data/management/podadmissionconfigurationtemplate_data.go

```diff
@@ -6,13 +6,36 @@ import (
 	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/types/config"
 	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var FeatureAppNS = []string{
+	"ingress-nginx",              // This is for Ingress, not feature app
+	"kube-system",                // Harvester, vSphere CPI, vSphere CSI
+	"cattle-system",              // AKS/GKE/EKS Operator, Webhook, System Upgrade Controller
+	"cattle-epinio-system",       // Epinio
+	"cattle-fleet-system",        // Fleet
+	"longhorn-system",            // Longhorn
+	"cattle-neuvector-system",    // Neuvector
+	"cattle-monitoring-system",   // Monitoring and Sub-charts
+	"rancher-alerting-drivers",   // Alert Driver
+	"cis-operator-system",        // CIS Benchmark
+	"cattle-csp-adapter-system",  // CSP Adapter
+	"cattle-externalip-system",   // External IP Webhook
+	"cattle-gatekeeper-system",   // Gatekeeper
```

Comment:
The latest version of the Gatekeeper chart [brings PSAs with it](https://github.com/rancher/charts/blob/8f6c52862117bdf7a6ccfed29a0c1847bfbd441e/charts/rancher-gatekeeper/101.1.0%2Bup3.10.0/values.yaml#L45-L50). If we wish to honour those perhaps we should not add the namespace for Gatekeeper here (unless it is the case that Gatekeeper will fail to install if it's not here).

Reply:
I think we still need to include the namespace because:
1. We support PSA in k8s 1.23 and above, so if users can use the PSACT in 1.23/1.24 cluster and install the old version of Gatekeeper, Gatekeeper can still be installed 
2. The latest version of Gatekeeper appends the PS labels to its namespace, which will overwrite the cluster-level default configuration for this namespace.